test/ExecutionTests.jl

```diff
@@ -283,7 +299,6 @@ io = IOBuffer()
 Profile.print(IOContext(io, :displaysize => (24, 200)))
 str = String(take!(io))
 @test occursin(r"BenchmarkTools(\.jl)?(/|\\)src(/|\\)execution\.jl:\d+; _run", str)
-@test !occursin(r"BenchmarkTools(\.jl)?(/|\\)src(/|\\)execution\.jl:\d+; warmup", str)
```

Comment:
Why does this disappear?

Reply:
This is left over from deleting warmup entirely from the package. It used to check that `@bprofile` wasn't profiling the warmup (important to avoid profiling the compiler).